content/docs/guides/continuous-delivery/_index.md

```diff
@@ -54,6 +54,10 @@ within a [generic CI/CD system]({{< relref "other.md" >}}).
         <img src="/logos/tech/ci-cd/jenkins.svg" alt="Jenkins">
         <h4 class="no-anchor">Jenkins</h4>
     </a>
+    <a href="{{< relref octopus-deploy >}}">
+        <img src="/logos/tech/ci-cd/octopus-deploy.svg" alt="Jenkins">
```

Comment:
Alt text should be "Octopus Deploy", I presume?

Also, just to double check, did you test this image when resizing the window various ways? Does it grow/shrink appropriately? For example, the Codefresh or GitHub Actions logo wouldn't behave unless I set an explicit width on them. (Presumably due to the canvas size of the `.svg` file itself.)

Reply:
Re: SVG resizing, NEVERMIND! I'll just check out the LIVE PREVIEW... And everything is AOK.

Reply:
> Alt text should be "Octopus Deploy", I presume?

Oops! Yes it should be. Will fix it.

> Also, just to double check, did you test this image when resizing the window various ways? Does it grow/shrink appropriately?

I simply checked that the image loaded and looks good, but didn't check if it resizes properly. I'll confirm that.

Reply:
I don't see any of the images growing/shrinking when adjusting the window size. Using the Mobile preview mode in FF adjusts the position of the logos correctly, but each image is still the same size from what I can tell. Is that expected?

Reply:
As long as it doesn't look "obviously broken", it's fine. (So images staying the same size is fine.)